socorro-cmd

```diff
@@ -65,58 +84,47 @@ class Group:
 COMMANDS = [
     Group(
         'Crash processing utilities', {
-            'add_crashid_to_queue': 'socorro.scripts.add_crashid_to_queue.main',
```

Comment:
Whoops--should have removed this a while back. That script doesn't exist anymore.